packages/eas-cli/src/commands/update/publish.ts

```diff
@@ -312,6 +312,26 @@ export default class UpdatePublish extends EasCommand {
       oldMessage = updatesToRepublishFilteredByPlatform[0].message ?? '';
       oldRuntimeVersion = updatesToRepublishFilteredByPlatform[0].runtimeVersion;
     } else {
+      if (!message && autoFlag) {
+        message = (await getVcsClient().getLastCommitMessageAsync())?.trim();
+      }
+
+      if (!message) {
+        const validationMessage = 'publish message may not be empty.';
+        if (jsonFlag) {
+          throw new Error(validationMessage);
+        }
+        ({ publishMessage: message } = await promptAsync({
+          type: 'text',
+          name: 'publishMessage',
+          message: `Please enter a publication message.`,
```

Comment:
```suggestion
          message: `Please enter an update message.`,
```
Since this will be applied to an update.